src/WidenImage.js

```diff
@@ -0,0 +1,86 @@
+import './WidenImage.css'
+import utilities from './Utilities'
+
+// If enabled, widened images will have thin red dashed border
+const enableDebugBorders = false
+
+const widenAncestors = (el) => {
+  while ((el = el.parentElement) && !el.classList.contains('content_block')) {
+    // Only widen if there was a width setting. Keeps changes minimal.
+    if (el.style.width) {
+      el.style.width = '100%'
+    }
+    if (el.style.maxWidth) {
+      el.style.maxWidth = '100%'
+    }
+    if (el.style.float) {
+      el.style.float = 'none'
+    }
+  }
+}
+
+const shouldWidenImage = (image) => {
```

Comment:
Hey quick question - any idea why this doesn't work?

    export default {
        _shouldWidenImage = shouldWidenImage,
        maybeWidenImage
    }